.github/label-sync.yml

```diff
@@ -199,24 +213,36 @@
       previousNames:
         - documentation
         - "type:documentation"
-    - name: "type:design"
+    - name: "type:design 📐"
+      color: efefef
+      previousNames:
+        - "type:design"
+    - name: "type:user-proofing 🛡"
       color: efefef
-    - name: "type:user-proofing"
+      previousNames:
+        - "type:user-proofing"
+    - name: "type:theme/plugin-conflict 💥"
       color: efefef
-    - name: "type:theme/plugin-conflict"
-      color: ebf7a0
+      previousNames:
+        - "type:theme/plugin-conflict"
     - name: "type:question ❓"
       color: efefef
       previousNames:
         - question
         - "type:question"
-    - name: "type-invalid 🦄"
+    - name: "type:invalid 🦄"
       color: efefef
       previousNames:
         - invalid
         - "type:invalid"
+        - "type-invalid 🦄"
     - name: "type:impacts-eventsmart"
       color: 00f2ff
+    # other labels
+    - name: "Epic 🤘"
+      color: 5d498c
+      previousNames:
+        - Epic
```

Comment:
hmm I think Epic is a label zenhub adds.  We should make sure this doesn't break how Zenhub does epics before making this change?

Reply:
kk I'll try it

Reply:
changed it and it appears to still work... but we can wait and see....
my guess is that GitHub uses a hidden internal identifier for the labels and the actual label text doesn't matter,
which would allow for localization, etc